### PR TITLE
Announce Configuration pages migration to Bootstrap

### PR DESCRIPTION
**Pull request by dmarcoux**
I had to create a new PR because we were not able to merge @saraycp's draft PR since she is the only one who can change the state of the PR.

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 70c4366204c5695c67d9d1063d09d541465879e8

https://deploy-preview-220--openbuildservice.netlify.com

**Comment by dmarcoux** [comment]
@openSUSE/obs-frontend Please review.

File: _posts/development/2019-03-03-revamping-configuration-ui.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+You can now see all architectures at a glance and enable/disable them all at once.
+Everything is done through AJAX, thus removing the need for an 'Update' button and saving you a click.
+
+<img src="/images/posts/revamping-configuration-ui/architectures.png" alt="Architectures Page" style="width: 80%;" class="center" />
+
+# Manage Users
+
+We have simplified the 'Actions' column of the datatable to only show the three most important actions: 'Edit User', 'Send Email to User' and 'Delete User'.
+You can still lock/confirm a user and make him/her an Admin, it's all on the 'Edit User' page.
```
Comment (hennevogel) [inline _posts/development/2019-03-03-revamping-configuration-ui.md]: The most important change is that this is now usable with a large number of users.

File: _posts/development/2019-03-03-revamping-configuration-ui.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+# Interconnect
+
+We replaced the unorthodox form and form-filler button combo by a list of default interconnects and a modal to add custom interconnects.
+
+<img src="/images/posts/revamping-configuration-ui/interconnect.png" alt="Interconnect Redesign" style="width: 80%;" class="center" />
+
+# And Much More
+
+We addressed some issues we identified and gathered from your feedback.
+Here’s a subset of what has been fixed and improved.
```
Comment (hennevogel) [inline _posts/development/2019-03-03-revamping-configuration-ui.md]: You could link to https://github.com/openSUSE/open-build-service/issues?q=is%3Aissue+label%3A%22Bootstrap+%3Arocket%3A%22+is%3Aclosed

File: _posts/development/2019-03-03-revamping-configuration-ui.md (hunk 1 above)
Comment (dmarcoux) [reply _posts/development/2019-03-03-revamping-configuration-ui.md]: I'm not sure how deep I should go with describing why it's now usable again. We already write about server side pagination later in the blog post.

Comment (dmarcoux) [reply _posts/development/2019-03-03-revamping-configuration-ui.md]: Let me know what you think about the paragraph I added above this one.

Comment (hennevogel) [reply _posts/development/2019-03-03-revamping-configuration-ui.md]: http://bremenbilder.de/Fotoland/A_PROFI/Bremen%20perspektivisch/Bremen%20perspektivisch%20Seiten/Bremen%20perspektivisch%20001.html

Comment (hennevogel) [reply _posts/development/2019-03-03-revamping-configuration-ui.md]: I like it :smile: remeber most users are power users and developers. no need to dumb things down too much.

**Comment by dmarcoux** [comment]
@hennevogel Please have another look

**Comment by dmarcoux** [comment]
I will wait for tomorrow, so everything is deployed.

**Comment by dmarcoux** [comment]
[Communicate the deliverable to the customers in a proper way](https://trello.com/c/7q8gWbEX/1060-communicate-the-deliverable-to-the-customers-in-a-proper-way)

**Pull request merged**